app/views/shared/_page_alert.html.haml

```diff
@@ -1,6 +1,6 @@
 - if ContentConfig.home_page_alert_html.present?
   .alert-cta
-    %h6= raw ContentConfig.home_page_alert_html
+    %h6= sanitize(@comment.body, tags: %w(strong em a i span), attributes: %w(href target))
```

Comment:
Should `@comment.body` be `ContentConfig.home_page_alert_html` ?

Also, if we have a standard set of allowed tags and attributes, I'd suggest defining these in a constant.

Reply:
Indeed, I must have been tired. Fixed it.